NxMapData.lua

```diff
@@ -1644,6 +1644,13 @@ Map.MapWorldInfo = {
 	[1648] = {
 		BaseMap = 1543
 	},
+	[1961] = {
+		Scale = 8.279166,
+		X = -1733.3332,
+		Y = -646.6666,
```

Comment:
These values are currently incorrect as the map does not show ingame with the current state of the PR.